jest.config.ts

```diff
@@ -30,9 +30,7 @@ const config: Config.InitialOptions = {
   coverageDirectory: 'coverage',
 
   // An array of regexp pattern strings used to skip coverage collection
-  // coveragePathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  coveragePathIgnorePatterns: ['\\/src\\/types\\.ts$'],
```

Comment:
The forward slashes do not need to be escaped
```suggestion
  coveragePathIgnorePatterns: ['/src/types\\.ts$'],
```

Reply:
Ah right, I suppose since we're not using `/.../` it doesn't matter.

Reply:
Changed in https://github.com/MetaMask/eth-json-rpc-infura/pull/58 as well.